documentation/modules/quickstart/proc-installing-product.adoc

```diff
@@ -83,3 +75,9 @@ kubectl create -f install/cluster-operator/031-RoleBinding-strimzi-cluster-opera
 --
 +
 The commands create role bindings that grant permission for the Cluster Operator to access the Kafka cluster.
+. Deploy the CRDs and role-based access control (RBAC) resources to manage the CRDs.
```

Comment:
@PaulRMellor Do we need an empty line before this and before line 64?

Reply:
Renders okay but it would make the source easier to read. Could you put lines as suggested by Jakub, @ShubhamRwt?